exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java

```diff
@@ -73,6 +73,10 @@ public ProfileWrapper(final QueryProfile profile, DrillConfig drillConfig) {
     final List<MajorFragmentProfile> majors = new ArrayList<>(profile.getFragmentProfileList());
     Collections.sort(majors, Comparators.majorId);
 
+    //Setting warning thresholds for performance-degrading queries (DRILL-6879)
```

Comment:
Do not use static methods, just pass drillConfig in the constructor for each of there classes and set warning threshold value during instance creation.

Reply:
Done